From 05fb5c083c6b083786b059db58f7626026dddffb Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 30 Aug 2024 00:19:01 -0400
Subject: [PATCH 3/5] Drop the nightly feature -- not for Debian

Nightly builds are irrelevant for distribution.  we're not going to
ship them, and they block our attempt to do tests with --all-features.
---
 Cargo.toml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 7b10977..dff28a5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -312,9 +312,4 @@ asm = [
     "md-5/asm",
 ]
 default = []
-nightly = [
-    "rsa/nightly",
-    "rand/nightly",
-    "num-bigint/nightly",
-]
 profile = ["gperftools"]
-- 
2.45.2

